src/dcap_provider.cpp

```diff
@@ -23,7 +23,9 @@
 #include <winsock.h>
 #endif
 
-#define MAX_URL_LENGTH 2000
+#define ENV_AZDCAP_BASE_URL "AZDCAP_BASE_CERT_URL"
+#define ENV_AZDCAP_CLIENT_ID "AZDCAP_CLIENT_ID"

```

Comment:
I am wondering the impact difference between those two environment vars.

Reply:
Can you elaborate? If only specified to unique set of known client id values, shouldn't affect anything else in the PCK Cache server's behavior drastically.

Reply:
I am sorry for not being more clear on my original question. I meant to ask what would end up being tested when different values are set in those two environment variables. I would be great to add comments in the code. Also, adding some log messages to give some hit on the values in use.

Reply:
Agreed with this. I added comments above the environment variable definitions outlining the intent of these values, and get_base_url() and get_client_id() have logging in-place indicating whether values from these environment variables have been set and being consumed.

Reply:
Thanks for addressing my comments.